src/components/Dialogs/OnlineDataQueryDialog/CatalogOnlineQueryComponent.tsx

```diff
@@ -144,7 +144,9 @@ export class CatalogQueryComponent extends React.Component {
                 </FormGroup>
                 <FormGroup inline={false} label="Center coordinates" disabled={disable}>
                     <Select
-                        items={Object.keys(SystemType).map(key => SystemType[key])}
+                        items={Object.keys(SystemType)
+                            .map(key => SystemType[key])
+                            .filter(sys => sys !== SystemType.Image)}
```

Comment:
maybe we can replace it with `items={Object.values(SystemType).filter(sys => sys !== SystemType.Image)}`